pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -28,6 +28,7 @@ import (
 
 // +genclient
 // +genclient:method=CreateAdminKubeconfigRequest,verb=create,subresource=adminkubeconfig,input=github.com/gardener/gardener/pkg/apis/authentication/v1alpha1.AdminKubeconfigRequest,result=github.com/gardener/gardener/pkg/apis/authentication/v1alpha1.AdminKubeconfigRequest
+// +genclient:method=CreateBinding,verb=create,subresource=binding,input=github.com/gardener/gardener/pkg/apis/core/v1alpha1.Binding,result=k8s.io/apimachinery/pkg/apis/meta/v1.Status
```

Comment:
From above: Actually, we are only interested in this tag here, so can we remove the `//+genclient` stuff from `types_binding.go` or are they needed there to make this line work?

Reply:
It is needed.

Reply:
OK, thanks for checking